reconcile/test/test_openshift_base.py

```diff
@@ -645,3 +645,18 @@ def test_determine_user_keys_for_access_not_implemented():
     auth = {"service": "not-implemented"}
     with pytest.raises(NotImplementedError):
         sut.determine_user_keys_for_access("cluster-name", [auth])
+
```

Comment:
add another one for `ns = {"delete": None}`

Reply:
https://github.com/app-sre/qontract-reconcile/pull/3010/commits/d79b6a4647c415eab7a929f040a8c6d0728efe58